### DISCUSSION TITLE
Background wallpaper for chats

### DISCUSSION THREAD

**Discussion by hardBSDk** [status-im/status-legacy#12618] [Ideas]
Implement an option to set custom background images for chats.

![Screenshot_20190131-191523](https://user-images.githubusercontent.com/55782009/126931881-e00bbbe0-d090-483a-b186-399e5c070160.png)

![Screenshot_20190131-191557](https://user-images.githubusercontent.com/55782009/126931887-66cfd0ad-37a5-404e-9a0b-3e9cce3a1988.png)

**Comment by churik** [comment]
moved to https://github.com/status-im/status-react/discussions/12619
